features-support/hooks/ui-automation.js

```diff
@@ -90,30 +91,41 @@ module.exports = function seleniumHooks() {
         .withCapabilities(capabilities)
         .build();
 
-      // Manage timeouts.
-      timeoutManager = driver.manage().timeouts();
-      timeoutManager.pageLoadTimeout(pageLoadTimeoutms);
-      timeoutManager.implicitlyWait(implicitlyWaitTimeoutms);
+      driver.getSession()
+          .then(function () {
+            // Manage timeouts.
+            timeoutManager = driver.manage().timeouts();
+            timeoutManager.pageLoadTimeout(pageLoadTimeoutms);
+            timeoutManager.implicitlyWait(implicitlyWaitTimeoutms);
 
+            driver.manage().window().maximize()
+                .then(function () {
+                  callback();
+                });
+
+          });
     } catch (error) {
```

Comment:
Okay. In general it is best practice to end a promise chain with error handling so it doesn't eat your useful error messages, but I can see that in this case it's not important.